src/services/IPFSService.ts

```diff
@@ -55,6 +55,11 @@ export default class IPFSService {
         method: 'GET',
         timeout,
       }),
+      axios.request({
+        url: 'https://davi.mypinata.cloud/ipfs/' + hash,
+        method: 'GET',
+        timeout,
+      }),
       axios.request({
         url: 'https://ipfs.io/ipfs/' + hash,
         method: 'GET',

```

Comment:
This can use some DRY'ing:

```typescript
const gatewayURLBaseList = [
  'https://davi.mypinata.cloud/ipfs',
  'https://dxgov.mypinata.cloud/ipfs',
  'https://ipfs.io/ipfs',
  'https://cloudflare-ipfs.com/ipfs',
  // more gateways 
]

const response = await Promise.any(
  gatewayURLBaseList.map((gatewayURLBase) =>
    axios.get(`${gatewayURLBase}/${hash}` , {
      timeout,
    })
  )
)
```